src/main/java/cn/nukkit/item/ItemMilkBucket.java

```diff
@@ -1,17 +1,27 @@
 package cn.nukkit.item;
 
-public class ItemMilkBucket extends ItemBucket {
+import cn.nukkit.Player;
+
+public class ItemMilkBucket extends ItemFood {
```

Comment:
Milk is quite unique, and it should extends the itembucket. You can implement its logical functions uniquely